telegram/bot/db.go

```diff
@@ -0,0 +1,96 @@
+package bot
+
+import (
+	"github.com/boltdb/bolt"
+)
+
+// IDReader reads ID by login from KV storage
+type IDReader interface {
+	ReadID(login string) string
+}
+
+// IDWriter writes ID by login to KV storage
+type IDWriter interface {
+	WriteID(login, id string) error
+}
+
+// DBCloser closes DB connection
+type DBCloser interface {
+	Close()
+}
+
+// DB implements DB interface
+type DB interface {
+	IDReader
+	IDWriter
+	DBCloser
+}
+
+type db struct {
+	db *bolt.DB
+}
+
+// NewDb connects to DB
+func NewDb(name string) DB {
+	instance := db{}
+	var err error
+	instance.db, err = bolt.Open(name, 0600, nil)
+	if err != nil {
+		logger.Fatal(err)
+	}
+	result := &instance
+	result.initBuckets([]string{"users"})
+	return result
+}
+
+func (d *db) Close() {
+	logger.Debug("DB closed")
+	d.db.Close()
+}
+
+func (d *db) initBuckets(buckets []string) {
```

Comment:
Some users could create their subscriptions in Moira, which are stored in redis. I see, you don't migrate those subscriptions to the new format,